dev/io.openliberty.org.jboss.resteasy.common/src/org/jboss/resteasy/specimpl/ResteasyHttpHeaders.java

```diff
@@ -24,7 +24,42 @@
 
+   private static final Map<String, List<MediaType>> mediaTypeCache;                                                                              // Liberty change
+   private static final Map<String, List<Locale>> languageCache;                                                                                  // Liberty change
+   static {                                                                                                                                       // Liberty change
+       Map<String, List<MediaType>> mtCache = new HashMap<>();                                                                                    // Liberty change
+       mtCache.put(MediaType.APPLICATION_ATOM_XML_TYPE.toString(), Collections.singletonList(MediaType.APPLICATION_ATOM_XML_TYPE));               // Liberty change
+       mtCache.put(MediaType.APPLICATION_FORM_URLENCODED_TYPE.toString(), Collections.singletonList(MediaType.APPLICATION_FORM_URLENCODED_TYPE)); // Liberty change
+       mtCache.put(MediaType.APPLICATION_JSON_PATCH_JSON_TYPE.toString(), Collections.singletonList(MediaType.APPLICATION_JSON_PATCH_JSON_TYPE)); // Liberty change
+       mtCache.put(MediaType.APPLICATION_JSON_TYPE.toString(), Collections.singletonList(MediaType.APPLICATION_JSON_TYPE));                       // Liberty change 
+       mtCache.put(MediaType.APPLICATION_OCTET_STREAM_TYPE.toString(), Collections.singletonList(MediaType.APPLICATION_OCTET_STREAM_TYPE));       // Liberty change
+       mtCache.put(MediaType.APPLICATION_SVG_XML_TYPE.toString(), Collections.singletonList(MediaType.APPLICATION_SVG_XML_TYPE));                 // Liberty change
+       mtCache.put(MediaType.APPLICATION_XHTML_XML_TYPE.toString(), Collections.singletonList(MediaType.APPLICATION_XHTML_XML_TYPE));             // Liberty change
+       mtCache.put(MediaType.APPLICATION_XML_TYPE.toString(), Collections.singletonList(MediaType.APPLICATION_XML_TYPE));                         // Liberty change
+       mtCache.put(MediaType.MULTIPART_FORM_DATA_TYPE.toString(), Collections.singletonList(MediaType.MULTIPART_FORM_DATA_TYPE));                 // Liberty change
+       mtCache.put(MediaType.SERVER_SENT_EVENTS_TYPE.toString(), Collections.singletonList(MediaType.SERVER_SENT_EVENTS_TYPE));                   // Liberty change
+       mtCache.put(MediaType.TEXT_HTML_TYPE.toString(), Collections.singletonList(MediaType.TEXT_HTML_TYPE));                                     // Liberty change
+       mtCache.put(MediaType.TEXT_PLAIN_TYPE.toString(), Collections.singletonList(MediaType.TEXT_PLAIN_TYPE));                                   // Liberty change
+       mtCache.put(MediaType.TEXT_XML_TYPE.toString(), Collections.singletonList(MediaType.TEXT_XML_TYPE));                                       // Liberty change
+       mtCache.put(MediaType.WILDCARD_TYPE.toString(), Collections.singletonList(MediaType.WILDCARD_TYPE));                                       // Liberty change
```

Comment:
Might as well use the constant you defined earlier in the class.

Reply:
yup, done